docs/index.rst

```diff
@@ -1,31 +1,32 @@
-Odoo development
-================
-
-Join our telegram group: https://t.me/odoo_talks
-
-.. toctree::
-   :maxdepth: 2
-
-   first-steps
-   dev/index
-   debugging/index
-   qa/index
-   migration/index
-   description/index
-   git/index
-   ci/index
-   odoo/index
-   admin/index
-   cd/index
-   maintenance/index
-   ide/index
-   remote-dev/index
-   other/index
-
-.. admonition:: Need our service?
-   :class: warning
-
-   For module development contact us by `email <mailto:it@it-projects.info>`__ or fill out `request form <https://www.it-projects.info/page/website.contactus>`__:
-
-   * it@it-projects.info
-   * https://www.it-projects.info/page/website.contactus
+Odoo development
+================
+
+Join our telegram group: https://t.me/odoo_talks
+
+.. toctree::
+   :maxdepth: 2
+
+   first-steps
+   dev/index
+   debugging/index
+   qa/index
+   migration/index
+   description/index
+   git/index
+   ci/index
+   odoo/index
+   pos/index

```

Comment:
Check errors on travis. You forgot to add pos/index.rst file
https://travis-ci.com/it-projects-llc/odoo-development/builds/112645118